content/docs/ref/runner.md

```diff
@@ -48,7 +50,10 @@ Any [generic option](/doc/ref) in addition to:
 - `--cloud-ssh-private=<key>`: Private SSH RSA key [default: *auto-generate
   throwaway key*]. Only supported on AWS and Azure; intended for debugging
   purposes.
-- `--cloud-aws-security-group=<...>`: AWS security group.
+- `--cloud-aws-security-group=<...>`:
+  [AWS security group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html).
```

Comment:
not sure if https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html is a more appropriate link?

Reply:
hard to say. Both looks similar